test/test_browser.py

```diff
@@ -3182,90 +3162,76 @@ def test_sdl2_pumpevents(self):
     ''')
     self.btest_exit('test_sdl2_pumpevents.c', args=['--pre-js', 'pre.js', '-sUSE_SDL=2'])
 
-  @no_wasm64('SDL2 + wasm64')
   def test_sdl2_timer(self):
     self.btest_exit('test_sdl2_timer.c', args=['-sUSE_SDL=2'])
 
-  @no_wasm64('SDL2 + wasm64')
   def test_sdl2_canvas_size(self):
     self.btest_exit('test_sdl2_canvas_size.c', args=['-sUSE_SDL=2'])
 
-  @no_wasm64('SDL2 + wasm64')
   @requires_graphics_hardware
   def test_sdl2_gl_read(self):
     # SDL, OpenGL, readPixels
     self.btest_exit('test_sdl2_gl_read.c', args=['-sUSE_SDL=2'])
 
-  @no_wasm64('SDL2 + wasm64')
+  @no_4gb('https://github.com/libsdl-org/SDL/issues/9052')
```

Comment:
Looks good, just checking this is intentional?

Reply:
Yes I think its the same issue.